features/ouders.feature

```diff
@@ -19,236 +19,234 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
-    En de ouder met burgerservicenummer 555550004 heeft haar geslacht en voornamen gewijzigd zodat de volgende persoonsgegevens in de registratie op haar persoonslijst staan:
-    | Categorie | Voornamen | Geslachtsaanduiding (04.10) |
-    | 1         | Charlotte | V                           |
-    | 51        | Karel     | M                           |
+    Gegeven de persoon met burgerservicenummer '000007895' heeft een ouder '1' met de volgende gegevens
+    | burgerservicenummer (01.20) | voornamen (02.10) | geslachtsaanduiding (04.10) |
+    | 000007896                   | Ellen             | V                           |
+    En de persoon heeft een ouder '2' met de volgende gegevens
+    | burgerservicenummer (01.20) | voornamen (02.10) | geslachtsaanduiding (04.10) |
+    | 000007897                   | Karel             | M                           |
+    En de ouder '2' is gecorrigeerd naar de volgende gegevens
+    | burgerservicenummer (01.20) | voornamen (02.10) | geslachtsaanduiding (04.10) |
+    | 000007897                   | Charlotte         | V                           |
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                                                                                  |
     | type                | RaadpleegMetBurgerservicenummer                                                         |
-    | burgerservicenummer | 555550003                                                                               |
+    | burgerservicenummer | 000007895                                                                               |
     | fields              | ouders.ouderAanduiding,ouders.burgerservicenummer,ouders.geslacht,ouders.naam.voornamen |
-    Dan heeft de ouder met ouderAanduiding '1' de volgende gegevens
+    Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
     | naam                  | waarde    |
+	| burgerservicenummer   | 000007896 |
+    | geslacht.code         | V         |
+    | geslacht.omschrijving | vrouw     |
+	| naam.voornamen        | Ellen     |
     | ouderAanduiding       | 1         |
-    | burgerservicenummer   | 555550004 |
+    En heeft de persoon een 'ouder' met de volgende gegevens
+    | naam                  | waarde    |
+	| burgerservicenummer   | 000007897 |
     | geslacht.code         | M         |
     | geslacht.omschrijving | man       |
-    En heeft de ouder met ouderAanduiding '1' de volgende 'naam' gegevens
-    | naam      | waarde |
-    | voornamen | Karel  |
+	| naam.voornamen        | Karel     |
+    | ouderAanduiding       | 2         |
 
 @gba
 Rule: Wanneer alleen gegevens in groep 81, 82, 83, 84, 85 en/of 86 zijn opgenomen en geen gegevens in groep 1, 2, 3, 4 of 62, dan wordt de ouder niet opgenomen
   Gebruik van de fields parameter heeft geen invloed op het leveren van een ouder
 
   @gba
```

Comment:
@gba tag moet weg, omdat in de Dan stap output van de proxy wordt getoond.

Reply:
Ga ik in een nieuwe PR aanpassen.